watertap/unit_models/pressure_changer.py

```diff
@@ -49,6 +143,36 @@ def calculate_scaling_factors(self):
             )
             iscale.constraint_scaling_transform(c, sf)
 
+        if hasattr(self, "bep_flow"):
+            if iscale.get_scaling_factor(self.bep_flow) is None:
+                iscale.set_scaling_factor(self.bep_flow, 10)
+
+        if hasattr(self, "bep_head"):
+            if iscale.get_scaling_factor(self.bep_head) is None:
+                iscale.set_scaling_factor(self.bep_head, 0.01)
+
+        if hasattr(self, "bep_eta"):
+            if iscale.get_scaling_factor(self.bep_eta) is None:
+                iscale.set_scaling_factor(self.bep_eta, 1)
+
+        if hasattr(self, "flow_ratio"):
+            if iscale.get_scaling_factor(self.flow_ratio) is None:
+                iscale.set_scaling_factor(self.flow_ratio, 1)
+
+        if hasattr(self, "efficiency_pump"):
+            if iscale.get_scaling_factor(self.efficiency_pump[0]) is None:
+                iscale.set_scaling_factor(self.efficiency_pump[0], 1)
+
+        # scale constraints
+
+        if hasattr(self, "flow_ratio_constraint"):
+            if iscale.get_scaling_factor(self.flow_ratio_constraint[0]) is None:
+                iscale.set_scaling_factor(self.flow_ratio_constraint[0], 1)
+
+        if hasattr(self, "eta_constraint"):
+            if iscale.get_scaling_factor(self.eta_constraint[0]) is None:
+                iscale.set_scaling_factor(self.eta_constraint[0], 1)
```

Comment:
I'm not sure we only want to scale the first time element. Perhaps we should remove `[0]` unless there is some reasoning behind this.

Reply:
Actually, I think this should be indexed and looped through by time